site/js/runner.js

```diff
@@ -1,8 +1,8 @@
 var Runner = {
     editor: null,
     enabled: false,
-    inputString: '',
-    runStep: '',
+    inputString: 'abba',
```

Comment:
In case of the Runner it's not easy to understand where the controller and where the view are. To me the Runner modules looks as a *view controller*. The `abba` default string was assigned in the `run` method of this module each time the user wants to make a run (event if the user used another string before). Now the previous string is preserved, and the user sees that default suggested string when the automaton runs the very first time and no other strings ware provided. 
Concluding, the default input string is set by a view controller (during its instantination).